.github/workflows/ci.yml

```diff
@@ -83,18 +83,6 @@ jobs:
           cargo xcheck ci esp32c6 --toolchain stable --no-lint --no-docs
           cargo xcheck ci esp32h2 --toolchain stable --no-lint --no-docs
 
-      - name: Semver-Check
-        shell: bash
-        # always invokes +esp internally
-        run: |
-          cargo xcheck semver-check --chips esp32 check
-          cargo xcheck semver-check --chips esp32s2 check
-          cargo xcheck semver-check --chips esp32s3 check
-          cargo xcheck semver-check --chips esp32c2 check
-          cargo xcheck semver-check --chips esp32c3 check
-          cargo xcheck semver-check --chips esp32c6 check
-          cargo xcheck semver-check --chips esp32h2 check
```

Comment:
Moved into its isolated workflow as it needs to be executed when every PR has a new label